agent/bench-scripts/test-bin/pbench-postprocess-tools

```diff
@@ -1 +0,0 @@
-pbench-start-tools
```

Comment:
All of this is to mimic how the actual `pbench-*-tools` files are provided so that we adding and removing mocks we are consistent.

This helped me track down exactly what is happening in the test system so that I could be sure when the mocks were removed the test was doing the right thing.

Reply:
OK, but...since the first thing the test does is remove all of these mocks, it doesn't seem like they needed to be modified in the end.